### DISCUSSION TITLE
Fill the entire graph area based on color_thresholds

### DISCUSSION THREAD

**Discussion by mzakharo** [kalkih/mini-graph-card#922] [Ideas]
Hello, would it be possible to fill the entire area of the graph based on color thresholds? Right now it only fills the area under the curve. Here is a grafana view for example: 
![image](https://user-images.githubusercontent.com/567867/219287046-23ca2dfd-058b-4c2d-a733-f39fecaa7be9.png)

Here is my best attempt with mini-graph-card:
![image](https://user-images.githubusercontent.com/567867/219287151-821f2117-0cb9-4f00-8e77-1ab23ae90c48.png)

**Comment by ildar170975** [comment]
Cannot be done by this card.

**Reply by mzakharo** [reply]
Is there any possibility of adding such functionality to this card?

**Comment by ildar170975** [comment]
Since this project is maintained by volunteers, no 100% guarantee that even a simpler functionality will be added. Let us hope. 
A detailed description (техническое задание) is needed to describe a logic.

**Reply by mzakharo** [reply]
[This](https://github.com/mzakharo/mini-graph-card/commit/e39b32b6b1d547cfb3a7f190c4fbae913425ea2c) somehow works for me. cheers. 
![image](https://user-images.githubusercontent.com/567867/219467524-56a21684-a20b-4376-9a5a-b7c88663721b.png)
